iothub_client/samples/iothub_client_device_cbor_sample/iothub_client_device_cbor_sample.c

```diff
@@ -0,0 +1,956 @@
+#include "iothub_message.h"
+
+#include "cbor.h"
+
+#define REPORTED_PROPERTY_BUFFER_SIZE 512
+#define TELEMETRY_MESSAGE_BUFFER_SIZE 128
+
+#define MAX_MESSAGE_COUNT 10
+#define TIMEOUT_RECEIVE_MS (30 * 1000)
+
+// Trusted Cert -- Turn on via build flag
+#ifdef SET_TRUSTED_CERT_IN_SAMPLES
+    #include "certs.h"
+#endif
+
+// Transport Layer Protocal -- Uncomment the protocol you wish to use.
+#define SAMPLE_MQTT
+//#define SAMPLE_MQTT_OVER_WEBSOCKETS
+
+#ifdef SAMPLE_MQTT
+    #include "iothubtransportmqtt.h"
+    static IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol = MQTT_Protocol;
+#elif defined SAMPLE_MQTT_OVER_WEBSOCKETS
+    #include "iothubtransportmqtt_websockets.h"
+    static IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol = MQTT_WebSocket_Protocol;
+#endif
+
+/**
+ * The content type system property for C2D and/or telemetry messages will appear as a key-value
+ * pair appended to the topic. They key is SDK-defined as `$.ct`, and URL-encoded as `%24.ct`. The
+ * value is application-defined and must be agreed upon between the device and service side
+ * applications. Examples for this value include `text/plain` and `application/json`. To
+ * demonstrate setting the content type system property, this sample uses `application/cbor`.
+ * See iothub_message.h for more system properties available to set for C2D and telemetry messaging.
+ */
+#define CONTENT_TYPE_C2D "application/cbor" // application-defined
+#define CONTENT_TYPE_TELEMETRY "application/cbor" // application-defined
+
+// Connection String -- Paste in the your iothub device connection string.
+static const char* connection_string = "[device connection string]";
+
+static IOTHUB_DEVICE_CLIENT_HANDLE iothub_client_handle;
+static IOTHUB_MESSAGE_HANDLE message_handle;
+
+// Telemetry
+static char* const telemetry_property_engine_temperature_name = "engine_temperature";
+static uint64_t telemetry_message_id = 0;
+
+// Device Twin Properties
+static uint8_t reported_property_buffer[REPORTED_PROPERTY_BUFFER_SIZE];
+static size_t reported_property_length;
+static char* twin_desired_property = "desired";
+
+static char* twin_property_manufacturer_name = "manufacturer";
+static char* twin_property_manufacturer_make_name = "make";
+static char* twin_property_manufacturer_model_name = "model";
+static char* twin_property_manufacturer_year_name = "year";
+static char* twin_property_state_name = "state";
+static char* twin_property_state_max_speed_name = "max_speed";
+static char* twin_property_state_software_version_name = "software_version";
+static char* twin_property_state_vanity_plate_name = "vanity_plate";
+static char* twin_property_change_oil_reminder_name = "change_oil_reminder";
+static char* twin_property_last_oil_change_date_name = "last_oil_change_date";
+
+typedef struct MANUFACTURER_TAG
+{
+    char* make;                 // reported property
+    char* model;                // reported property
+    uint64_t year;              // reported property
+} Manufacturer;
```

Comment:
please update these to match the case of the `_TAG`
```suggestion
} MANUFACTURER;
```

and below for State and Car